soh/soh/SohMenuBar.cpp

```diff
@@ -902,6 +902,8 @@ void DrawEnhancementsMenu() {
             UIWidgets::Tooltip("Always shows dungeon entrance icons on the minimap");
             UIWidgets::PaddedEnhancementCheckbox("Show Gauntlets in First Person", "gFPSGauntlets", true, false);
             UIWidgets::Tooltip("Renders Gauntlets when using the Bow and Hookshot like in OOT3D");
+            UIWidgets::PaddedEnhancementCheckbox("Enemy Health Bars", "gEnemyHealthBar", true, false);
+            UIWidgets::Tooltip("Renders a health bar above enemies when Z-Targeted");
```

Comment:
I'm torn between this being added under "Enhancements > Graphics", or "Enhancements > Gameplay"

Looking for feedback on placement.

Reply:
I would argue for gameplay since we're effectively changing the state of the game in a minor manner.

Reply:
Agreed with Enhancements > Gameplay